gradle.properties

```diff
@@ -21,7 +21,7 @@ org.gradle.jvmargs=-Xmx1G
 	# The Curseforge versions "names" or ids for the main branch (comma separated: 1.16.4,1.16.5)
 	# This is needed because CF uses too vague names for prereleases and release candidates
 	# Can also be the version ID directly coming from https://minecraft.curseforge.com/api/game/versions?token=[API_TOKEN]
-	release-curse-versions = Minecraft 1.20:1.20.2
+	release-curse-versions = Minecraft 1.20.3:1.20.4
```

Comment:
Oops, didn't notice this isn't correct (it follows some weird curseforge versioning where the prefix is the minor only and suffix is with patch), easy fix though